src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -264,6 +219,9 @@ public function escapeValue($value)
         return $value;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function unescapeValue($value)
```

Comment:
I'm wondering if this function should also be moved in the ``ParameterResolver``.